js/react/pages/payment/success.jsx

```diff
@@ -83,9 +85,12 @@ function PaymentSuccess() {
 						<p>
 							Your payment was successful, and we are currently processing your order.
 							As a result of this donation, you have earned{' '}
-							<a href="/profile/donations" target="_blank">
-								{10 + Math.ceil(successInfo?.amount * 2)} donation karma points.
-							</a>
+							<b>{10 + Math.ceil(successInfo?.amount * 2)}</b> donation karma points.
+							<PopOver
+								text={Translations.DONATION_HISTORY.karmaPoints.popOverText}
+								isImgProvided={true}
+								imgSrc="/img/tshirt-2023.png"
+							/>
```

Comment:
please check the changes in master, I've reworked this component a bit since a few fields were redundant